Philippines/salamat.md

```diff
@@ -11,7 +11,7 @@ Buksan ang [pinto] (bagong simula/bagong_simula.md).
 Pahinga muna sa pag-aaral. Manood ng [movie] (movie/movie.md)
 
 Nakapag pahinga na. Salamat sa mga [pelikula] (movie/movie.md).
-Ngayon ay ating sundan ang mga kaganapan sa Pilipinas. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.